### ISSUE TITLE
How to placing AR object at specific  longitude and  latitude?

### ISSUE THREAD

**Issue by a100447739** [mapbox/mapbox-unity-sdk#618]
I am using WorldScaleAR scene and wondering how to place AR object at  specific  longitude and  latitude?

**Comment by lauri-rustanius** [comment]
Hi! You can use the GeoToWorldPosition method in the AbstractMap class.

**Comment by atripathi-mb** [comment]
@a100447739  Use the `GeoToWorldPosition` method on the `AbstractMap` to get positions in Map space. You can use these positions to spawn your objects. 

Or if you want to have custom data, you might want to create a custom dataset using Mapbox Studio and create a tileset to consume in Unity. Let me know which option solves your problem best and we can make sure you have all the information you need.

**Comment by campeezjesse** [comment]
Could this work on a tabletop size AR map as well? 
 I am searching for a way to add spatial sound anchored to specific coordinates. Both on the World scale and tabletop maps.

**Comment by atripathi-mb** [comment]
@campeezjesse yes, it should work.

**Comment by josefsHD** [comment]
Hello! I'm coming to unity from unreal so very new to the scripting side. I have added the abstract map script to a cube as a component. I tried changing the general settings Lat/long to position the cube to a real world location about 10 metres from my window, but when deployed to the phone it is at the player origin. Which part of the AbstractMap script do I add the GeoToWorldPosition element to? Or is it already present in the script and I just have to change some numbers? Thanks in advance!
